.devcontainer/devcontainer.json

```diff
@@ -1,22 +1,32 @@
 {
-    "extensions": [
-		"golang.go"
-	],
-	"build": {
-		"dockerfile": "Dockerfile",
-		"args": {
-			"VARIANT": "1.18"
-		}
+	"image": "mcr.microsoft.com/devcontainers/go:1.18",
+	"features": {
+		"ghcr.io/devcontainers/features/github-cli:1": {
```

Comment:
I don't think we need the `github-cli` feature. Users of this codespace can just run `make` to build their own `bin/gh` from source. It might be potentially confusing to have another, older version of `gh` installed system-wide.